docs/pages/guides/permissions.md

```diff
@@ -26,30 +28,54 @@ You can find the full list of available properties in [Apple's InfoPlistKeyRefer
-- Apps using dangerous or signature permissions without valid reasons _may be rejected by Google_. Make sure you follow the [Android permissions best practices](https://developer.android.com/training/permissions/usage-notes) when submitting your app.
-- By default, the permissions implied by the modules you installed are added to the **AndroidManifest.xml** at build time. To exclude permissions, you have to define the `android.permissions` manifest property.
+Apps using dangerous or signature permissions without valid reasons _may be rejected by Google_. Make sure you follow the [Android permissions best practices](https://developer.android.com/training/permissions/usage-notes) when submitting your app.
 
 ### Bare workflow
 
 In the bare workflow, permissions are controlled in your project **AndroidManifest.xml**.
 
-Some Expo and React Native modules include permissions by default. If you use `expo-location`, for example, both the `ACCESS_COARSE_LOCATION` and `ACCESS_FINE_LOCATION` are implied and added to your app's permissions automatically. To limit the permissions your managed workflow app requires, add them them to a [list of explicitly excluded permissions](#excluding-android-permissions).
+Some Expo and React Native modules include permissions by default. For example, if you use `expo-location`, the `ACCESS_COARSE_LOCATION`, `ACCESS_FINE_LOCATION` and `FOREGROUND_SERVICE` are added to your app's permissions automatically. To limit the permissions your bare workflow app requires, see [Excluding Android Permissions](#excluding-android-permissions).
 
 Apps using dangerous or signature permissions without valid reasons _may be rejected by Google_. Make sure you follow the [Android permissions best practices](https://developer.android.com/training/permissions/usage-notes) when submitting your app.
 
-#### Excluding Android permissions
+### Excluding Android permissions
+
+When adding Expo and other React Native modules to your project, certain Android permissions might be implied automatically. The modules should only add relevant permissions **required** to use the module. However, sometimes you may want to remove some of these permissions.
+
+For example, using `expo-location` module, the `ACCESS_COARSE_LOCATION`, `ACCESS_FINE_LOCATION` and `FOREGROUND_SERVICE` are added to your app's permissions automatically. If your application doesn't need access to the precise location, you can exclude the `ACCESS_FINE_LOCATION` permission.
+
+When you are building your app with [EAS Build](/build/introduction/), you can use a [Config Plugin](/guides/config-plugins/#using-a-plugin-in-your-app) in **app.json** or **app.config.json** to exclude a permission. In the example config below, the `ACCESS_FINE_LOCATION` is omitted when the app is built using EAS:
+
+```json
+{
+  "expo": {
+    "plugins": [
+      "expo-location": {
+        "withBlockedPermission": ["ACCESS_FINE_LOCATION"]
+      }
```

Comment:
Oh wait! This one is incorrect, we don't have the `withBlockedPermission` option in the `expo-location`'s config plugin: https://github.com/expo/expo/blob/main/packages/expo-location/plugin/src/withLocation.ts#L24-L30.

And it should be in this format:
```json
{
  "expo": {
    "plugins": [
      "expo-location",
      { ...options }
    ]
  }
}
```

Reply:
I also see your point in using the blocked android permissions, but we have to do either one of this:
1. Make the `withBlockedAndroidPermissions` a separate config plugin
2. Add the option, like `withoutAndroidPermissions`, to each config plugin

Reply:
I think that is worth doing. However, I don't have a strong opinion on which approach will be the best. I'm okay with both solutions.

Reply:
Sorry for not getting back here 😅 

I am failing to understand to exclude permission using a Config plugin vs the [docs currently recommending](https://docs.expo.dev/distribution/app-stores/#android-permissions) which is to add `android.blockedPermissions` array. I think we need to have "best practice" approach if there are multiple ways to do it in the docs (as to not create any confusion).